pkg/yang/yang.go

```diff
@@ -552,13 +557,40 @@ func (s *AnyXML) NName() string         { return s.Name }
 func (s *AnyXML) Statement() *Statement { return s.Source }
 func (s *AnyXML) Exts() []*Statement    { return s.Extensions }
 
+// An AnyData is defined in: http://tools.ietf.org/html/rfc7950#section-7.10
+//
+// AnyData are only expected in YANG 1.1 modules (those with a
+// "yang-version 1.1;" statement in the module).
+type AnyData struct {
```

Comment:
Alternative: as `anydata` have the same sub-statements as `anyxml` in NETCONF 1.0/1.1, this could alternatively be written simply as;

```go
type AnyData AnyXML

func (AnyData) Kind() string { return "anydata" }
func (s *AnyData) ParentNode() Node { return s.Parent }
...
```

Do you have a preference for one or the other?